### PR TITLE
Car icon

### PR DESCRIPTION
**Pull request by lieblichaaron**
## ℹ️ Change description:
Nothing here

Screenshot 2023-02-27 at 19 02 37

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React, {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx:1]: fix warnings

File: examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx [hunk 2]
```diff
@@ -77,8 +74,14 @@ const AvailabilityVehicle = ({
       coordinate={locationAnimated}
       anchor={{ x: 0.5, y: 0.40 }}
     >
-      <SvgIcon Svg={carIcon} height={48} width={48} fill={primaryColor} />
+      <SvgIcon
+        Svg={carIcon}
+        height={48}
+        width={48}
+        style={{ transform: [{ rotate: `${location.bearing}deg` }], color: vehicleColor }}
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx:95]: does it the same as the control center?

**Review by avivbarniv — APPROVED** [review]

File: examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx (hunk 2 above)
Comment (lieblichaaron) [reply examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx:95]: what do you mean? cc doesn't have vehicle icons

File: examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-import { Platform, View } from 'react-native';
-import styled from 'styled-components';
+import {
+  Platform,
+} from 'react-native';
 import { MarkerAnimated, AnimatedRegion } from 'react-native-maps';
 import { Context as ThemeContext } from '../../context/theme';
 import SvgIcon from '../SvgIcon';
-import carIcon from '../../assets/map/car.svg';
+import carIcon from '../../assets/map/autofleet_car_icon.svg';
```
Comment (OmerGery) [inline examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx]: ../../assets/map/Autofleet_Car_Icon.svg

Comment (lieblichaaron) [reply examples/client/Locomotion/src/Components/AvailabilityVehicle/index.tsx]: why?

**Pull request merged**